test/models/placement_test.rb

```diff
@@ -9,6 +9,32 @@ class PlacementTest < ActiveSupport::TestCase
     ActsAsTenant.current_tenant = tenants(:toughbyte_tenant)
   end
 
+  test "should not allow not disqualified placements to have disqualify_reason" do
+    placement = placements(:sam_golang_replied)
+
+    assert_not placement.disqualified?
+    assert_not placement.disqualify_reason
+
+    placement.update(disqualify_reason: disqualify_reasons(:no_reply_toughbyte))
```

Comment:
On testing validation you can just assign attribute. To make tests faster we should avoid writing to db if it's not required